src/components/connection_handler/CMakeLists.txt

```diff
@@ -28,6 +28,9 @@
 # ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
 
+if (${CMAKE_SYSTEM_NAME} MATCHES "Windows")
```

Comment:
ConnectionHandler have no need for OpenSSL.
It can use Security interfaces only